tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -951,6 +958,7 @@ const reducer = createReducer(
     let nextCardStepIndexMap = {...state.cardStepIndex};
     let nextLinkedTimeSelection = state.linkedTimeSelection;
     let nextStepSelectorEnabled = state.stepSelectorEnabled;
+    let linkedTimeRangeEnabled = state.rangeSelectionEnabled;
```

Comment:
Can you give me a little bit more color on why we need to enable linked time when range is checked?

Reply:
This is actually a miss from when I split this apart from #5962. I'm surprised this didn't throw a type error :thinking: